Makefile

```
@@ -3,11 +3,20 @@ MAINPACKAGE=./cmd/manager
 GOENV=GOOS=linux GOARCH=amd64 CGO_ENABLED=0
 GOFLAGS=-gcflags="all=-trimpath=${GOPATH}" -asmflags="all=-trimpath=${GOPATH}"
 
+TESTTARGETS := github.com/openshift/dedicated-admin-operator/pkg/controller/namespace github.com/openshift/dedicated-admin-operator/pkg/dedicatedadmin
```

```suggestion
TESTTARGETS := $(shell go list -e ./... | egrep -v "/(vendor)/")
```

This will auto generate a list of non-vendor targets to test.

---

Nice one!